ArduCopter/mode_loiter.cpp

```diff
@@ -152,21 +150,18 @@ void ModeLoiter::run()
         break;
 
     case AltHold_Landed_Ground_Idle:
-
-        attitude_control->reset_rate_controller_I_terms();
```

Comment:
I'm not sure I understand why this statement was deleted.  This is inconsistent with Copter 3.6 landed state.  I agree that adding the I term reset below to the landed_pre_takeoff state.

Reply:
@bnsgeyer it's been moved to AltHold_Landed_Pre_Takeoff? The sequencing has subtely changed in 4.0 I believe which was why I was seeing large iterm build up while the copter was not actually flying

Reply:
@andyp1per I think it should be in the landed state as well

Reply:
I defer to @lthall 😄 - I'm not convinced the naming of the states is completely consistent with actually what is going on. All I can say is what was there before for AltHold resulted in one nasty crash and several hairy moments.

Reply:
I think it was deleted because there's a fall though so it gets run a moment later in the state right below.  I.e. so now it's run in both states..

Reply:
Ok so that’s what I was missing. Now it makes sense